Terminal.Gui/Terminal.Gui.csproj

```diff
@@ -124,9 +125,6 @@
     <GitRepositoryRemoteName>upstream</GitRepositoryRemoteName>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <EnableSourceLink>true</EnableSourceLink>
-    <!--<DebugType>Embedded</DebugType>-->
     <Authors>Miguel de Icaza, Tig Kindel (@tig), @BDisp</Authors>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
VS likely stuck these down here the last time they were changed in the UI. They belong up top with the other build settings.